src/openfermion/ops/representations/doci_hamiltonian_test.py

```diff
@@ -23,15 +24,20 @@
 from openfermion.ops.representations.doci_hamiltonian import (
     DOCIHamiltonian, get_tensors_from_doci, get_projected_integrals_from_doci,
     get_doci_from_integrals)
+from openfermion import (get_fermion_operator, InteractionOperator, \
+                         normal_ordered)
+
+numpy.set_printoptions(linewidth=2000, threshold=sys.maxsize)
```

Comment:
this can be remove right? Since the print statements are removed below.

Reply:
Yes, these print options can be removed since they was used mostly for debugging purposes. However, print statements still exist in the test_fermionic_hamiltonian_from_integrals, and are also useful if the assert statements raise an error in the construction of an object of interest.